priv/repo/migrations/20220310174843_drop_expired_matches.exs

```diff
@@ -0,0 +1,7 @@
+defmodule T.Repo.Migrations.DropExpiredMatches do
+  use Ecto.Migration
+
+  def change do
+    drop table(:expired_matches)
```

Comment:
I suggest against dropping tables in migrations. Instead the app should be made able to work without them. And after it is, the tables can be dropped out-of-band (direct sql query to postgres), and migrations that created them -- removed.

Reply:
@ruslandoga okay, do I understand it correctly that upon release of the version with this PR I need to:
1) connect to prod db
2) drop these tables
3) create a separate PR removing migrations that created these tables
?

Reply:
No, this PR just needs to stop using the expired and archived matches tables.